docs/src/api.md

```diff
@@ -0,0 +1,168 @@
+| ---                        | ---                                                                                                                               | ---                                                             |
+| `pointwise_loglikelihoods` | [`DynamicPPL.pointwise_loglikelihoods`](@extref)                                                                                  | Compute log likelihoods for each sample in a chain              |
+| `generated_quantities`     | [`DynamicPPL.generated_quantities`](@extref)                                                                                      | Calculate additional quantities defined in a model              |
+| `logprior`                 | [`DynamicPPL.logprior`](@extref)                                                                                                  | Compute log prior probability                                   |
+| `logjoint`                 | [`DynamicPPL.logjoint`](@extref)                                                                                                  | Compute log joint probability                                   |
+| `LogDensityFunction`       | [`DynamicPPL.LogDensityFunction`](@extref)                                                                                        | Wrap a Turing model to satisfy LogDensityFunctions.jl interface |
+| `condition`                | [`AbstractPPL.condition`](@extref)                                                                                                | Condition a model on data                                       |
+| `decondition`              | [`AbstractPPL.decondition`](@extref)                                                                                              | Remove conditioning on data                                     |
+| `conditioned`              | [`DynamicPPL.conditioned`](@extref)                                                                                               | Return the conditioned values of a model                        |
+| `fix`                      | [`DynamicPPL.fix`](@extref)                                                                                                       | Fix the value of a variable                                     |
+| `unfix`                    | [`DynamicPPL.unfix`](@extref)                                                                                                     | Unfix the value of a variable                                   |
+| `OrderedDict`              | [`OrderedCollections.OrderedDict`](https://juliacollections.github.io/OrderedCollections.jl/dev/ordered_containers/#OrderedDicts) | An ordered dictionary                                           |
+
+### Extra re-exports from Bijectors
+
+Note that Bijectors itself does not export `ordered`.
+
+```@docs
+Bijectors.ordered
+```
```

Comment:
Is there a particular reason why Bijectors doesn't export `ordered`, and Turing does?

That seems counterintuitive to me because if it's important enough for Turing to export it, surely it must be important enough for Bijectors to export it.

If we export it from Bijectors, then we can add it to the docs there and then just link to it here.